pallets/vector/src/benchmarking.rs

```diff
@@ -164,13 +164,18 @@ pub const ROTATE_VK: &str = r#"{"vk_json":{
     ]
 }}"#;
 
+// Generated with SP1 Helios https://github.com/succinctlabs/sp1-helios/blob/main/README.md
+// cargo prove key —-elf (sp1 helios elf) in SP1 Helios
+const SP1_VERIFICATION_KEY: [u8; 32] =
+	hex!("00788ce8dc2970920a3d3c072c8c07843d15f1307a53b3dd31b113c3e71c28e8");
```

Comment:
Note: On the latest `main` for SP1 Helios, we've created reproducible builds with Docker. The latest verification key is `0x1b9571a44d20ab19380ef04e5e01052f3ca90a9a7394ad7b2c8b7a5f4705067b`.

You can reproduce with `cargo run --bin vkey --release` on the latest `sp1-helios` main.

Reply:
This is just for the benchmarks, the vk will be added after the release.